helix-loader/src/grammar.rs

```diff
@@ -89,15 +89,106 @@ pub fn fetch_grammars() -> Result<()> {
     let mut grammars = get_grammar_configs()?;
     grammars.retain(|grammar| !matches!(grammar.source, GrammarSource::Local { .. }));
 
-    run_parallel(grammars, fetch_grammar, "fetch")
+    print!("Fetching {} grammars", grammars.len());
```

Comment:
`print!` doesn't flush stdout so this doesn't end up printing until the " ... done" later. I think it would be ok to turn this into a `println!` and remove the done part.

Reply:
Done